site/src/request_handlers/github.rs

```diff
@@ -51,8 +51,7 @@ async fn handle_push(ctxt: Arc<SiteCtxt>, push: github::Push) -> ServerResult<gi
         let rollup_merges = commits
             .iter()
             .rev()
-            .skip(1) // skip the head commit
-            .take_while(|c| c.message.starts_with("Rollup merge of "));
+            .filter(|c| c.message.starts_with("Rollup merge of #"));
```

Comment:
I am a *little* worried about what happens when we rollup a subtree merge or something similar that itself contains Rollup merge ... -- particularly since today we suffer a complete queue stoppage if rustup-toolchain-install-master fails until the DB is manually cleaned.

But I think for the most part those PRs aren't rolled up, so this is probably OK for now. We can revisit and try to get some more advanced filtering at a later point if needed.